cables.py

```diff
@@ -64,35 +64,31 @@ def get_cables_on_pages(list_of_page_elements):
         if not cables_list:
             logging.info(f'There are no cables on page {page_name}')
             continue
-        logging.info(f'page_name: {page_name}')
-        for cable_dict in cables_list: 
-            logging.info(f'(cable_dict):{cable_dict}')
```

Comment:
This might be very verbose, but it helps me identify cables that are actually lines without needed information.